### DISCUSSION TITLE
Distinguish between different types of Bicycle Parking (especially lockers and other enclosed parking)

### DISCUSSION THREAD

**Discussion by SIVAH-AKASH** [organicmaps/organicmaps#9629] [Ideas]
It would be great to distinguish b/w bicycle parking which is located outside and bicycle parking located in an enclosed area (i.e. lockers, shed and building types). The latter is more secure and usually suitable for long term storage.

**Comment by patepelo** [comment]
Related to [#6236](https://github.com/organicmaps/organicmaps/issues/6236)
